samples/snippets/src/main/java/pubsub/CreatePullSubscriptionExample.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2020 Google LLC
```

Comment:
Are these new for 2020? I think the general guidance on copyright dates for older code is to keep the original year (and add -2020 if you want).

Reply:
I referenced the old samples, updated most of them to 2016, except for DLQ samples.